ballerina/receiver.bal

```diff
@@ -117,7 +117,7 @@ public isolated client class MessageReceiver {
     isolated remote function complete(@display {label: "Message"} Message message)
                                         returns Error? {
         if message?.lockToken.toString() != DEFAULT_MESSAGE_LOCK_TOKEN {
-            return complete(self, message?.lockToken.toString());
+            return complete(self, message);
         }
         return createError("Failed to complete message with ID " + message?.messageId.toString());
```

Comment:
```suggestion
        return createError(string `"Failed to complete message with ID: ${message?.messageId}"`);
```

Reply:
Sure, Addressed in [d68d5f2](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/196/commits/d68d5f20b64a6b0bb9f12a693a9b53510cb2ef25)